treeherder/webapp/api/urls.py

```diff
@@ -162,6 +162,11 @@
         performance_data.PerformanceSummary.as_view(),
         name='performance-summary',
     ),
+    re_path(
+        r'^performance/alerttasks/(?P<id>\d+)/$',
```

Comment:
Could you call this `alertsummary-tasks` to better reflect what we're pulling from? The ID that you're passing is for an alert summary rather than a single alert, and the `-` makes it a bit easier to read :)